web/src/stream_edit.js

```diff
@@ -102,6 +105,7 @@ export function open_edit_panel_for_row(stream_row) {
 export function open_edit_panel_empty() {
     const tab_key = stream_settings_ui.get_active_data().$tabs.first().attr("data-tab-key");
     $(".stream-row.active").removeClass("active");
+    $("#subscription_overlay .right").removeClass("show");
```

Comment:
I had to fix this selector too; @aparna-bhatt please consider the feedback about things like bad selectors across the whole change, not just in the one place it was mentioned.